src/components/FriendPage/FriendPage.js

```diff
@@ -6,23 +6,33 @@ import { useEffect, useState } from 'react';
 const FriendPage = ({ id }) => {
   const friends = useSelector(state => state.friends)
   const [friend, setFriend] = useState({})
+  const [showForm, setForm] = useState(false)
 
   useEffect(() => {
-    console.log(friends)
     setFriend(friends.find(friend => parseInt(friend.id) === id))
-  }, [friends])
+  }, [friends, id])
+
+  const displayForm = () => {
+    return (
+      <form>
+        <input type='text' placeholder='Gift idea'/>
+        <button>Save idea</button>
```

Comment:
Might be nice to have an additional button like nevermind or something in case they no longer want to add a gift to be able to close out of the form.